### PR TITLE
typo fixes

### PR DESCRIPTION
**Pull request by EvanM27**
update readme

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+- [General info](#general-info)
+- [Technologies](#technologies)
+- [Setup](#setup)
+
+## General info
+
+- This project is the back-end for the mount recorder application.
+- It stores users, mounts, and which mounts users own.
+- It returns data when recieving SQL statements.
```
Comment (sofia819) [inline README.md]: This is a pretty wrong explanation of what the API does.
- It does not `store users, mounts, and which mounts users own`. That is the database's job.
- It does not receive `SQL statement`. It receives API requests.


Just put this:
```
- The project is the back-end API used in [Mount Recorder Client](https://github.com/sofia819/mount_recorder_client).
- On each API request from the client, the corresponding operations are executed and data is returned when necessary.
```

File: README.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+## Technologies
+
+Project is created with:
```
Comment (sofia819) [inline README.md]: Remove

File: README.md [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+
+Language
+
+- JavaScript
+
+Environment
+
+- Node.js
```
Comment (sofia819) [inline README.md]:
```
- [Node.js](https://nodejs.org/en/)
```

File: README.md [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+
+Framework
+
+- Express.js
```
Comment (sofia819) [inline README.md]:
```
- [Express.js](https://expressjs.com/)
```

File: README.md [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+
+Libraries
+
+- bcrypt
+- dotenv
+- cors
+- pg-promise
```
Comment (sofia819) [inline README.md]:
```
- [bcrypt](https://github.com/kelektiv/node.bcrypt.js)
- [dotenv](https://github.com/motdotla/dotenv)
- [cors](https://github.com/expressjs/cors)
- [pg-promise](https://github.com/vitaly-t/pg-promise)
```

File: README.md (hunk 1 above)
Comment (EvanM27) [reply README.md]: I'm honestly amazed I forgot something so foundational and critical. The corrections have been made.

File: README.md (hunk 2 above)
Comment (EvanM27) [reply README.md]: Removed

File: README.md (hunk 3 above)
Comment (EvanM27) [reply README.md]: Link added

File: README.md (hunk 4 above)
Comment (EvanM27) [reply README.md]: link added

File: README.md (hunk 5 above)
Comment (EvanM27) [reply README.md]: links added

**Review by sofia819 — APPROVED** [review]

**Pull request merged**